src/python/WMCore/Services/Service.py

```diff
@@ -294,11 +294,22 @@ def getData(self, cachefile, url, inputdata=None, incoming_headers=None,
                     cachefile.write(data)
                     cachefile.seek(0, 0)  # return to beginning of file
                 else:
-                    with open(cachefile, 'w') as f:
-                        if isinstance(data, dict) or isinstance(data, list):
-                            f.write(json.dumps(data))
-                        else:
-                            f.write(data)
+                    # Ensure that the path for /tmp file is created before
+                    # creating the file
+                    pathfile=cachefile.split("/")
+                    pathfile="/"+"/".join(pathfile[1:5])
```

Comment:
```suggestion
                    pathfile="/"+"/".join(pathfile[1:-1])
```
this would be a bit more reliable, e. g. the path for container's caché is a bit different, so the harcoded `5` doesn't work

Reply:
Thanks for the suggestion @germanfgv. I added the changes. Lets keep using this PR to test and deploy the T0 agents but Im still investigating the root of this issue.